app/views/layouts/main.html

```diff
@@ -74,7 +74,7 @@
 
 {% block bodyEnd %}
   <!-- GOV.UK Prototype Kit {{releaseVersion}} -->
-  {{ data | log }}
+  {# {{ data | log }} #} {# This causes an error on template pages for some reason. #}
```

Comment:
When viewing a template in the `manage-prototype` view, using the `log` filter seems to cause an error as it's not defined. There's probably a way to fix this but the simpler solution was to just comment this line out for now.